assets/js/components/KeyMetrics/ChipTabGroup/index.js

```diff
@@ -49,12 +50,19 @@ import { BREAKPOINT_SMALL, useBreakpoint } from '../../../hooks/useBreakpoint';
 import CheckMark from '../../../../svg/icons/check-2.svg';
 import { MODULES_ANALYTICS_4 } from '../../../modules/analytics-4/datastore/constants';
 import { CORE_UI } from '../../../googlesitekit/datastore/ui/constants';
+import { CORE_USER } from '../../../googlesitekit/datastore/user/constants';
+import StarFill from '../../../../svg/icons/star-fill.svg';
 
 const currentSelectionGroup = {
 	SLUG: KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG,
 	LABEL: __( 'Current selection', 'google-site-kit' ),
 };
 
+const suggestedGroup = {
+	SLUG: KEY_METRICS_SUGGESTED_GROUP_SLUG,
+	LABEL: __( 'Suggested', 'google-site-kit' ),
+};
```

Comment:
Let's mov this and `currentSelectionGroup` to `constants.js` for consistency with other groups.